Dockerfile

```diff
@@ -74,7 +74,10 @@ RUN \
     rm    storage/framework/views/* 2> /dev/null || true && \
     rm    storage/logs/* 2> /dev/null || true && \
     chown -R www-data:www-data /var/www/html/Lychee && \
-    chmod -R g+ws storage/image-jobs storage/livewire-tmp && \
+    chmod -R g+ws storage/image-jobs && \
+    @chmod -R g+ws storage/livewire-tmp 2> /dev/null || true && \
+    @chmod -R g+ws storage/image-tmp 2> /dev/null || true && \
+    @chmod -R g+ws storage/extract-jobs 2> /dev/null || true && \

```

Comment:
Why `@`? I'm not sure we really need the `/dev/null` parts as even the `|| true` should only be needed until the v6 release.

Reply:
good spot :)

Reply:
Fixed : [fe34506](https://github.com/LycheeOrg/Lychee-Docker/pull/208/commits/fe3450651bb19948a5a18043515b20eaa5cb8276)